eternalcore/build.gradle.kts

```diff
@@ -69,7 +69,7 @@ bukkit {
     name = "EternalCore"
     description = "All the most important server functions in one!"
     version = "${project.version}"
-    depend = listOf("PlaceholderAPI")
+    softDepend = listOf("PlaceholderAPI")
```

Comment:
Rollczi zrobił to w sposób taki że eternalcore wywali się bez PlaceholderAPI

Reply:
no tak, tylko ja pisałem że przy zmianie na softDepend -> HookService który będzie sprawdzał czy jest PAPI na serwerze i będzie rejestrowało PAPIPlaceholderReplacer

Reply:
wtedy EC nie jest zależny od papi, jak ktoś chce to używa jak nie to nie